package.json

```diff
@@ -104,7 +100,7 @@
     "react-dom": "17.0.2",
     "react-test-renderer": "^17.0.2",
     "regenerator-runtime": "^0.13.1",
-    "semantic-release": "^17.4.3",
+    "semantic-release": "19.0.5",
```

Comment:
where's "@semantic-release/git": "^10.0.1" ?

Reply:
This lib was not present here in the project so I chose not to include it. Seeking to understand the behavior of this lib I checked the catho-components and although the lib is included there in the project apparently it is not working. I believe it is a case of correcting it there in catho-components and including it here in quantum as well, considering that the changelog directly in the commit tree helps to contextualize the independent project on github